prepWork/databaseModel.md

```diff
@@ -0,0 +1,32 @@
+
+# Database Model Schema
+
+Below is our Database Model. We will be storing data in our model from our API.
```

Comment:
Please remove as we won't be storing nft data in our database - will fetch from chain.

Reply:
We need to have a database diagram, lets talk more about what we would use in place of this